### ISSUE TITLE
Create a batch HGVS checker on LOVD.nl

### ISSUE THREAD

**Issue by ifokkema** [LOVDnl/LOVD3#573]
Depends on #566.

The internal LOVD function `lovd_getVariantInfo()` is currently being rebuilt by @loeswerkman which adds detailed HGVS checks and informative error messages in the case that variants are not HGVS-conformant. It would be a waste to not use this code also more broadly. Especially since tools like VariantValidator and Mutalyzer seem to sometimes give very random-looking and unclear error messages, it would be beneficial to create a batch HGVS syntax checker for our LOVD website using our new function that provides useful error messages to guide users in the right direction to fix their non-HGVS variant descriptions.

- [ ] Create a good set of correct and incorrect variant descriptions (e.g., our `lovd_getVariantInfo()` unit test set).
- [ ] Run this set through the Variant Validator batch checker.
- [ ] Run this set through the Mutalyzer batch checker.
- [ ] Compare our error messages with theirs. Document this well, so we can use it for a detailed report.
- [ ] See where we can improve if needed.
- [ ] Write a batch checker for LOVD.nl that does better. May require a separate issue or more details in this issue.
- [ ] Will be worth disseminating on social media and the HUGO newsletter.
